test/testHelpers.hpp

```diff
@@ -307,29 +307,37 @@ template<typename T>
 bool compareArraysRMSD(dim_t data_size, T *gold, T *data, double tolerance)
 {
     double accum  = 0.0;
-    double maxion = FLT_MAX;//(double)std::numeric_limits<T>::lowest();
+    double maxion = FLT_MIN;//(double)std::numeric_limits<T>::lowest();
```

Comment:
Are you sure you don't mean `-FLT_MAX`. FLT_MIN will give you a value close to 0.

Reply:
True, I have question of previous to mine. Why are the `std::numeric_limits` commented in the first place ? They point to the right values and we don't need to bother about signs.